Test_Automation_Framework/EPAM_TestAutomation_Tests/Basic2_EpamSiteNavigateTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Epam_TestAutomation_BusinessLogic.PageObjects.Pages;
+using Epam_TestAutomation_Core.Browser;
+using NUnit.Framework;
+using OpenQA.Selenium;
+
+namespace Epam_TestAutomation_Tests
+{
+    [TestFixture]
+    public class Basic2_EpamSiteNavigateTests : BaseTest
+    {
+        private SearchResultsPage _searchResultsPage;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _searchResultsPage = new SearchResultsPage();
+        }
+
+        [Test]
+        public void CheckTheListOfCountriesInTheCareerButtonTest()
+        {
+            var countries = new List<string> { "AMERICAS", "EMEA", "APAC" };
+
+            _searchResultsPage.CareerButton.Click();
+            var ourOfficesCareer = _searchResultsPage.OurOfficesCareer;
+            var careerElements = ourOfficesCareer.FindElements(By.XPath(".//div"))
```

Comment:
move webElement to the Epam_TestAutomation_BusinessLogic.PageObjects.Pages 
Fix for all tests